src/main/java/liquibase/ext/spanner/datatype/MediumIntTypeSpanner.java

```diff
@@ -13,13 +13,15 @@
  */
 package liquibase.ext.spanner.datatype;
 
+import com.google.cloud.spanner.Dialect;
 import liquibase.database.Database;
 import liquibase.datatype.DatabaseDataType;
 import liquibase.datatype.core.MediumIntType;
 import liquibase.ext.spanner.ICloudSpanner;
 
 public class MediumIntTypeSpanner extends MediumIntType {
   private static final DatabaseDataType INT64 = new DatabaseDataType("INT64");
+  private static final DatabaseDataType BIGINT = new DatabaseDataType("BIGINT");
```

Comment:
Spanner PostgreSQL supports the `int` type as an alias for `bigint`. We should use `int` here to ensure that the correct type is being created if in the future the `int` type is fully supported.

```suggestion
  private static final DatabaseDataType BIGINT = new DatabaseDataType("int");
```